### PR TITLE
adds issue templates to the COSI controller

### PR DESCRIPTION
**Pull request by krishchow**
- This PR adds issue templates for the COSI controller - this will be vital as this project progresses so we have a solid framework for tracking issues and requests

### CODE REVIEW

**Comment by msau42** [comment]
Should we also have a template for reporting security issues?

**Comment by krishchow** [comment]
Thanks for the heads up @msau42, I added the template for the security policy now

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**How to reproduce this bug (as minimally and precisely as possible)**:
+
+**Anything else relevant for this bug report?**:
+
+**Environment**:
+
+- Kubernetes version (use `kubectl version`), please list client and server:
+- Controller version (provide the release tag or commit hash):
+- Cloud provider or hardware configuration:
```
Comment (wlan0) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Add a line for Provisioner Name and Version

Comment (krishchow) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Added

**Pull request merged**